gbench/clblast/blas3/gemm.cpp

```diff
@@ -0,0 +1,160 @@
+/***************************************************************************
+ *
+ *  @license
+ *  Copyright (C) 2016 Codeplay Software Limited
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  For your convenience, a copy of the License has been included in this
+ *  repository.
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  SYCL-BLAS: BLAS implementation using SYCL
+ *
+ *  @filename gemm.cpp
+ *
+ **************************************************************************/
+
+#include "range.hpp"
+#include "utils.hpp"
+
+
+template <typename scalar_t>
+void BM_Gemm(benchmark::State& state) {
+  // Standard test setup.
+  char const* t_a = benchmark::utils::from_transpose_enum(
+      static_cast<benchmark::utils::Transposition>(state.range(0)));
+  char const* t_b = benchmark::utils::from_transpose_enum(
+      static_cast<benchmark::utils::Transposition>(state.range(1)));
+  const index_t m = static_cast<index_t>(state.range(2));
+  const index_t k = static_cast<index_t>(state.range(3));
+  const index_t n = static_cast<index_t>(state.range(4));
+/* "c"*/
```

Comment:
Could you explain this commented "c"? If not, please remove it.

Reply:
I have no idea why it's there, thanks for catching that